readthedocs/api/v3/views.py

```diff
@@ -472,37 +523,41 @@ def perform_create(self, serializer):
         serializer.save()
 
 
-class OrganizationsViewSet(
-    APIv3Settings,
-    NestedViewSetMixin,
-    OrganizationQuerySetMixin,
-    ReadOnlyModelViewSet,
-):
-    model = Organization
-    lookup_field = "slug"
-    lookup_url_kwarg = "organization_slug"
-    queryset = Organization.objects.all()
-    serializer_class = OrganizationSerializer
-    permission_classes = [
-        IsAuthenticated & (UserOrganizationsListing | IsOrganizationAdminMember)
-    ]
-    permit_list_expands = [
-        "projects",
-        "teams",
-        "teams.members",
-    ]
-
-    def get_view_name(self):
-        return f"Organizations {self.suffix}"
-
-    def get_queryset(self):
-        # Allow hitting ``/api/v3/organizations/`` to list their own organizations
-        if self.basename == "organizations" and self.action == "list":
-            # We force returning ``Organization`` objects here because it's
-            # under the ``organizations`` view.
-            return self.admin_organizations(self.request.user)
-
-        return super().get_queryset()
+# NOTE: old attempt to implement /api/v3/organizations/
+# Currently, it's commented because we are not allowing hitting this endpoint.
+# We can recover this implementation once we are ready.
```

Comment:
I think it's better if we create an issue with this code instead of having this buried in our codebase.